dotnetv3/cross-service/PhotoAnalyzerApp/PhotoAnalyzerApp/Views/Home/Index.cshtml

```diff
@@ -0,0 +1,22 @@
+@{
+    ViewData["Title"] = "Home Page";
+}
+<head>
+    <script src="https://code.jquery.com/jquery-1.11.1.min.js"></script>
+    <script src="https://code.jquery.com/ui/1.11.1/jquery-ui.min.js"></script>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" />
+</head>
+
+<h1 class="display-4">AWS Photo Analyzer Application</h1>
+<p>
+    The AWS Photo Analyzer application is a sample application that uses the Amazon Rekognition Service as well as other AWS Services and the AWS SDK for .NET.
+    Analyzing nature photographs has never been easier! Simply perform these steps:
```

Comment:
Update last sentence: 
Perform the following steps:

Reply:
fixed